docs/sources/flow/reference/components/otelcol.processor.attributes.md

```diff
@@ -0,0 +1,582 @@
+---
+title: otelcol.processor.attributes
+---
+
+# otelcol.processor.attributes
+
+`otelcol.processor.attributes` accepts telemetry data from other `otelcol`
+components and modifies attributes of a span, log, or metric.
+It also supports the ability to filter and match input data to determine if 
+it should be included or excluded for attribute modifications.
+
+> **NOTE**: `otelcol.processor.attributes` is a wrapper over the upstream
+> OpenTelemetry Collector `attributes` processor. Bug reports or feature requests
+> will be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.processor.attributes` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+otelcol.processor.attributes "LABEL" {
+  output {
+    metrics = [...]
+    logs    = [...]
+    traces  = [...]
+  }
+}
+```
+
+## Arguments
+
+`otelcol.processor.attributes` doesn't support any arguments and is configured fully
+through inner blocks.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.processor.attributes`:
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+output | [output][] | Configures where to send received telemetry data. | yes
+action | [action][] | Actions to take on the attributes of incoming metrics/logs/traces. | no
+include | [include/exclude] | Filter for data to be included to this processor's actions. | no
+include > regexp | [regexp] | Regex cache settings. | no
+include > attribute | [attribute] | A list of attributes to match against. | no
+include > resource | [resource] | A list of items to match the resources against. | no
+include > library | [library] | A list of items to match the implementation library against. | no
+include > log_severity_number | [library] | How to match against a log record's SeverityNumber, if defined. | no
+exclude | [include/exclude] | Filter for data to be excluded from this processor's actions | no
+exclude > regexp | [regexp] | Regex cache settings. | no
+exclude > attribute | [attribute] | A list of attributes to match against. | no
+exclude > resource | [resource] | A list of items to match the resources against. | no
+exclude > library | [library] | A list of items to match the implementation library against. | no
+exclude > log_severity_number | [library] | How to match against a log record's SeverityNumber, if defined. | no
+
```

Comment:
You're missing the sentence which explains the meaning of the `>` symbol: https://github.com/grafana/agent/blob/main/docs/developer/writing-flow-component-documentation.md#blocks